tests/test_optim.py

```diff
@@ -407,7 +407,7 @@ def test_optim_uncerts(backend, source, spec, mu):
         return_uncertainties=True,
     )
     assert result.shape[1] == 2
-    assert pytest.approx([0.0, 0.26418431]) == pyhf.tensorlib.tolist(result[:, 1])
+    assert pytest.approx([0.26418431, 0.0]) == pyhf.tensorlib.tolist(result[:, 1])
```

Comment:
this just changes the order

Reply:
i guess can we add one more assert here on the order of the parameters in the model, just so we can keep track moving forward as well? This will future-proof in case it somehow changes again.